app/models/organisation.rb

```diff
@@ -25,6 +25,7 @@ class Organisation < ActiveRecord::Base
   validates_presence_of :whitehall_slug
   validates_uniqueness_of :whitehall_slug
   validates_presence_of :title
+  validates_presence_of :content_id
```

Comment:
I've added a migration which removes the orphaned organisations and adds a unique constraint to the database. I'm always hesitant with adding `validates_uniqueness_of` as it means ActiveRecord does a query every time you save to get all the values for that field.